public/app/features/integrated-alerting/components/EmptyBlock/EmptyBlock.tsx

```diff
@@ -0,0 +1,14 @@
+import { useStyles } from '@grafana/ui';
+import React from 'react';
+import { FC } from 'react';
+import { getStyles } from './EmptyBlock.styles';
+
+export const EmptyBlock: FC<{ dataQa?: string }> = ({ children, dataQa }) => {
+  const style = useStyles(getStyles);
```

Comment:
Please create an interface and put it in a file for the types.

Reply:
Got it